.pre-commit-config.yaml

```diff
@@ -23,7 +23,7 @@ repos:
           - "--max-line-length=110"
           - "--max-complexity=18"
           - "--select=B,C,E,F,W,T4,B9"
-          - "--ignore=W503,E203"
+          - "--ignore=W503,E203,E501"
```

Comment:
What is this ignoring?

Reply:
length of line. I added a link in a comment that was longer than the maximum length

Reply:
Can you ignore the individual line, or break it up? Would rather keep the check.